pyodata/v2/model.py

```diff
@@ -781,6 +780,10 @@ def __init__(self, namespace):
             self.associations = dict()
             self.association_sets = dict()
 
+            #generated collections for ease of lookup (e.g. function import return type)
+            self._collections_entity_types = dict()
+            self._collections_complex_types = dict() #TODO
```

Comment:
This will be in second commit, so first draft is shorter to read. Also, I am modifying relevant test, so should be in separate commits anyway.

Reply:
Not sure which second commit you have on mind. The standard approach is to just not add what's not necessary.

Reply:
I know.. But it was marked as WIP PR,.. and I did not want to do all the necessary work, only to be rejected... so wanted some preliminary review if the idea is valid half-way.